### PR TITLE
Add GetDynamicActorAnimation

### PR DESCRIPTION
**Pull request by IstuntmanI**
I didn't test it, but it looks alright to me.

### CODE REVIEW

**Comment by samp-incognito** [comment]
Looks good.

File: streamer.inc [hunk 1]
```diff
@@ -357,6 +357,7 @@ native IsValidDynamicActor(STREAMER_TAG_ACTOR actorid);
 native IsDynamicActorStreamedIn(STREAMER_TAG_ACTOR actorid, forplayerid);
 native GetDynamicActorVirtualWorld(STREAMER_TAG_ACTOR actorid);
 native SetDynamicActorVirtualWorld(STREAMER_TAG_ACTOR actorid, vworld);
+native GetDynamicActorAnimation(STREAMER_TAG_ACTOR actorid, animlib[], animname[], &Float:fdelta, &loop, &lockx, &locky, &freeze, &time, maxanimlib = sizeof animlib, maxanimname = sizeof animname);
```
Comment (karimcambridge) [inline streamer.inc]: const should be added to these arrays

Comment (IstuntmanI) [inline streamer.inc]: They aren't const, they are modified by the plugin. const should be added only when that parameter won't be modified under any circumstance.

Comment (karimcambridge) [inline streamer.inc]: LOL sorry I haven't slept in a while I just noticed

**Pull request merged**